src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -868,6 +868,19 @@ protected function _BuildFilterSection()
 
             $filterContent .= '<div class="form-group mr-2 typeahead-relative">';
 
+            $filterContent .= '<script>
+                function resetPagingAndSearch(event) {
```

Comment:
The group tabled may be used multiple times on one page - the Method should be moved into the appropriate js file (`chameleon/blackbox/javascript/table.js`)

Reply:
This "in-line" with the existing old JS that is partly replaced here.

Especially here is a need for a (static) parameter to that JS: $this->listName

Can it be mulitply on one page? I think every table is in its own iframe?

Also it does work with a single id (searchLookup) for the search field.

Reply:
you can get the list name by using `${this}` (you will find the list name as the input field `_listName` AND as the name and id of the containing form.

The old JS would not cause a collision since it did not define a function - the new code does. So we need to take care of that issue.

The core may currently only use one instance of the list table on one page - but the list table itself is setup in suche a way, that multiple instances of it can be used on one page. I see no reason to limit this as a side effect - especially when the problem can be solved so easily.

Reply:
Wasn't easy (javascript!) but I moved it to a file and made it dynamic.